characters/Character.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can you rename this to something more specific pls? Maybe block.svg

Reply:
Sure! np